### PR TITLE
Move 'Case Groups' menu item

### PR DESCRIPTION
**Pull request by orangejenny**
http://manage.dimagi.com/default.asp?185253

@NoahCarnahan

### CODE REVIEW

File: corehq/apps/hqwebapp/models.py [hunk 1]
```diff
@@ -929,6 +916,21 @@ def contacts_urls(self):
                  'url': reverse('chat_contacts', args=[self.domain])}
             )
 
+        if self.couch_user.can_edit_data():
+            from corehq.apps.data_interfaces.views \
```
Comment (dimagimon) [inline corehq/apps/hqwebapp/models.py]: F401 'ArchiveFormView' imported but unused

**Comment by czue** [comment]
looks good.

@dimagi/product any need to communicate anything about this change or good to release? feel free to merge if no need to announce.

**Comment by orangejenny** [comment]
@dimagi/product ping

**Comment by sheelio** [comment]
Just update the wiki page, so good to merge.

**Comment by orangejenny** [comment]
@sheelio great, thanks!

**Pull request merged**